myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -4,155 +4,159 @@
-	    <div class="day-label">TUE</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings tuesday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">WED</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings wednesday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">THU</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings thursday start_time end_time }}
-	    </div>
-	</div>
-
-	<div {{#if has_6_days}}class="six-day"{{else}}class="five-day"{{/if}}>
-	    <div class="day-label">FRI</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings friday start_time end_time }}
-	    </div>
-	</div>
-
-	{{#if has_6_days}}
-	<div class="six-day">
-	    <div class="day-label">SAT</div>
-	    <div class="day-col hours-{{total_hours}}">
-	        {{ show_card_days_meetings saturday start_time end_time }}
-	    </div>
-	</div>
-	{{/if}}
-    </div>
-    {{/if}}
-    <!-- end week view -->
-
-    <!-- begin - if there are meetings with times to be arranged -->
-    {{#if courses_meeting_tbd }}
-    <div class="courseTBD" aria-hidden="true">
-        {{#each courses_meeting_tbd }}
-	    <div class="pull-left course-box">
-	        <div class="c{{color_id}} course-info"><a href="/{{#if ../is_instructor}}teaching{{else}}academics{{/if}}/#{{curriculum}}-{{course_number}}-{{section_id}}" style="color: white;" class="show_section_card">{{curriculum}} {{course_number}}&nbsp;{{section_id}}</a></div>
-	        <div style="font-size:.8em; color:#555; text-align:center; margin-top:3px;">Room TBD</div>
-	    </div>
-        {{/each}}
-    </div>
-    {{/if}}
-    <!-- end - if there are meetings with times to be arranged -->
 
+    <h3 id="quarter-info" >
+            {{ucfirst quarter}} {{year}} Schedule
+            {{#if summer_term}}
+            <span class="shortTitle">{{summer_term}}</span>
+            {{/if}}
+    </h3>
+    <div><p class="sr-only"><a href="#CourseCards">Skip the visual schedule and go to course cards</a></p></div>
+    <div aria-hidden="true" role="presentation"><!-- Wrap the content under aria-hidden -->
+        <div id="schedule-tabs" class="schedule-period-module">
+    		<ul class="nav nav-pills schedule-period-list">
+            {{#each schedule_periods }}
+            <li class="{{#equal ../active_period_id @key}} active{{/equal }}"><a class="schedule-period-anchor" href="#" data-period_id="{{ @key }}"><span class="schedule-period">{{#equal "finals" @key}}Finals {{ else }}{{ toMonthDay start_date }} - {{ toMonthDay end_date }} {{/equal }}</span></a></li>
+            {{/each }}
+    		</ul>
+        </div>
+		<h4 class="sr-only">Your {{ucfirst quarter}} {{year}} class schedule</h4>
```

Comment:
It doesn't really make sense here to have a `<h4 class="sr-only">Your {{ucfirst quarter}} {{year}} class schedule</h4>` inside an `aria-hidden="true"` <div>.

You're hiding this entire content so you can remove all the "sr-only" stuff from it.